index.html

```diff
@@ -17,21 +18,31 @@
 <body>
 
  <!-- navigation code here -->
+  <div class="title-nav">
+    <h1>SpaceWalkers</h1>
+    <nav>
+      <a href="#">Home</a>
+      <a href="#">About Us</a>
+      <a href="#">Blog</a>
+      <a href="#">Contact</a>
+      <a href="#">Sign In</a>
+    </nav>
+  </div>
 
   <div class="container home">
 
     <header class="intro">
        <!-- image code here -->
-
+      <img src="img/main-header.png">
       <h2>Lift Off</h2>
     
       <p>As we got further and further away, it [the Earth] diminished in size. Finally it shrank to the size of a marble, the most beautiful you can imagine. That beautiful, warm, living object looked so fragile, so delicate, that if you touched it with a finger it would fall apart. Seeing this has to change a man.</p>
     </header>
 
     <section class="space-images">
         <img src="img/blast.png" class="blast" alt="Rocket blasting off">
-        <img src="img/sky.png" class="sky" alt="Space sky image">
-        <img src="img/launch.png" class="launch" alt="Launching rocket">
+        <img class="hide" src="img/sky.png" class="sky" alt="Space sky image">
+        <img class="hide" src="img/launch.png" class="launch" alt="Launching rocket">
```

Comment:
I am guessing you didn't notice the class applied to each of these after the `src` path. I like seeing that you only added the classes where you needed them (i.e. these 2 images, but they already had classes you could target so it is redundant,